### ISSUE TITLE
Deploying index.html that connects to flask services

### ISSUE THREAD

**Issue by rodrigoSantamaria** [tiangolo/uwsgi-nginx-flask-docker#70]
Hi,
Thanks for the container, it's quite handy! :)
I'm trying to connect my app/static/index.html to the services I deploy in app/main.py
My index and my services work ok.

In my index.html there is a javascript (so to speak) connect(A), which is basically a call to one of my services, being A the hostname were my services are deployed. In my docker container, A=127.0.0.1 but when someone runs it, I want it to point to B (her particular hostname).
I tried during docker build with:
`RUN sed -i 's/127.0.0.1:2750/'`hostname`'.'`dnsdomainname`'/' /app/static/index.html`
But it will require new builds for each person that wants to use it in his host, and anyways hostname does not give the proper context inside the docker image and dnsdomainname does not exist on it.

Is there any option in docker run or in your container for it?  Is this a weird way of doing thing? I find it quite usual, and I'd prefer keep things in a single container.

Thanks!!

**Comment by tiangolo** [comment]
In your `index.html` you probably have a path like `http://127.0.0.1/api/users` or something similar, you can change it to `/api/users`. 

As your `index.html` is being served by the same container, it has the same hostname (IP) and port, so you can use absolute paths without necessarily adding the "scheme" (`http`) and tdhe hostname (`127.0.0.1`). 

So, if you have something like:

```html
<a href="http://127.0.0.1/api/users">Download the users data</a>
```

you can change that to:

```html
<a href="/api/users">Download the users data</a>
```

**Comment by rodrigoSantamaria** [comment]
Thanks for the quick answer! Unfortunately your solution did not worked for an ajax request, which is why I use in my javascript, or at least I wasn't able to make it work.

I found a workaround that might be useful for someone else.
I copied the `tiangolo entrypoint.sh` to my root folder (the one that contains the app folder) and added a line like this:
`sed -i 's/127.0.0.1:2750/'$SERVERNAME'/' /app/static/index.html`
In my Dockerfile I added:
`COPY ./entrypoint.sh /entrypoint.sh`

And when I run it i added a -e parameter with my hostname:
`docker run -it --rm -p 80:80 -e "SERVERNAME=myhostname.domain" myTag`

Cheers!
